src/app/code/community/Shopgate/Cloudapi/Model/Api2/Customers/Email/Validator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+class Shopgate_Cloudapi_Model_Api2_Customers_Email_Validator extends Shopgate_Cloudapi_Model_Api2_Validator
+{
+    const FIELD_EMAIL = 'email';
+
+    /**
+     * Validates the incoming email address data
+     *
+     * @inheritdoc
+     * @throws Zend_Validate_Exception
+     */
+    public function isValidData(array $data, $partial = false)
+    {
+        if (empty($data[self::FIELD_EMAIL])) {
```

Comment:
I don't think this case is possible

Reply:
The handling of this error is triggered by one of my tests.
"new email is empty"
return is
{
    "messages": {
        "error": [
            {
                "path": "email",
                "messages": [
                    "The email address cannot be empty.",
                    "Invalid email address \"\"."
                ],
                "message": "",
                "code": 400
            }
        ]
    }
}

Reply:
ah yes, I thought that it would invalidate if no params are passed, but the param is passed, just empty